cognite/powerops/custom_modules/power_model_v1/data_models/containers/SHOPResult.container.yaml

```diff
@@ -0,0 +1,100 @@
+space: '{{powerops_models}}'
+externalId: SHOPResult
+name: SHOPResult
+usedFor: node
+properties:
+  timeseries:
+    type:
+      list: true
+      type: timeseries
+    nullable: true
+    autoIncrement: false
+    name: timeseries
+    description: Timeseries from output from shop
+  tsDirect1:
+    type:
+      list: false
+      type: timeseries
+    nullable: true
+    autoIncrement: false
+    name: tsDirect1
+    description: A timeseries from the shop result (used in more specific extensions of a shop result)
+  tsDirect2:
+    type:
+      list: false
+      type: timeseries
+    nullable: true
+    autoIncrement: false
+    name: tsDirect2
+    description: A timeseries from the shop result (used in more specific extensions of a shop result)
+  tsDirect3:
+    type:
+      list: false
+      type: timeseries
+    nullable: true
+    autoIncrement: false
+    name: tsDirect3
+    description: A timeseries from the shop result (used in more specific extensions of a shop result)
+  tsDirectList1:
+    type:
+      list: true
+      type: timeseries
+    nullable: true
+    autoIncrement: false
+    name: tsDirectList1
+    description: A timeseries from the shop result (used in more specific extensions of a shop result)
+  tsDirectList2:
+    type:
+      list: true
+      type: timeseries
+    nullable: true
+    autoIncrement: false
+    name: tsDirectList2
+    description: A timeseries from the shop result (used in more specific extensions of a shop result)
```

Comment:
If we can use an extended time series type (see comment above), we may not need these...
Anyways, does it make sense to call the lists ts**Direct**List1 and 2, or would just tsList1 and 2 be better?

Reply:
Good idea. I'm working on an example for that